test/parallel/test-repl-syntax-error-stack.js

```diff
@@ -17,7 +17,7 @@ common.ArrayStream.prototype.write = function(output) {
 
 const putIn = new common.ArrayStream();
 repl.start('', putIn);
-let file = path.resolve(__dirname, '../fixtures/syntax/bad_syntax');
+let file = path.resolve(common.fixturesDir, 'syntax/bad_syntax');
```

Comment:
Again, could you split up `syntax/bad_syntax` as separate params and use `path.join()`